charts/timescaledb-single/.helmignore

```diff
@@ -1,5 +1,4 @@
 *.png
-kustomize/*
```

Comment:
removed this so the kustomize script can work when packaged with the chart
Whitelisting with ! seems to be broken (https://github.com/helm/helm/issues/3622)
So I couldn't add `!kustomize/example` to this

Reply:
Hmm, that's actually an issue.
The whole point is to avoid having secrets be part of the Helm deployment, if these directories are included, than the secrets are again part of the deployment.

Reply:
Ah, my bad we *do* exclude it from Helm, which I think is fine.